papr/papr-trigger.py

```diff
@@ -72,7 +72,8 @@ def generate_papr_pod(args):
                     "name": "papr",
                     "image": "172.30.1.1:5000/projectatomic-ci/papr",
                     "imagePullPolicy": "Always",
-                    "args": ["--debug", "--repo", args.repo],
+                    "args": ["--debug", "runtest", "--conf",
```

Comment:
I should probably call this out here. This switches from the old way of running `papr` to the new way in the Python rewrite.